classes/03class/exercises/c03-compose01/mrcsmonteiro/docker-compose.yaml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+services:
+  db:
+    image: mysql:5.7
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: RootP4ssword
+      MYSQL_DATABASE: devops_academy
+      MYSQL_USER: db_user
+      MYSQL_PASSWORD: 4pp_n4meP4sswrd0!
+  web:
+    build: .
+    ports:
+      - "8100:80"
+    volumes:
+      - ~/Docker/src:/var/www/html
+    restart: always
+    environment:
+      DB_USER: db_user
+      DB_PASS: 4pp_n4meP4sswrd0!
+      DB_HOST: db
```

Comment:
Good usage of the internal Docker network, using `db` as the host without exposing ports